logstage/logstage-rendering-circe/src/main/scala/izumi/logstage/api/rendering/json/LogstageCirceRenderingPolicy.scala

```diff
@@ -17,6 +17,11 @@ class LogstageCirceRenderingPolicy(
 
   import LogstageCirceRenderingPolicy._
 
+  protected val EventKey = "event"
+  protected val ContextKey = "context"
+  protected val MetaKey = "meta"
+  protected val TextKey = "text"
```

Comment:
```suggestion
  protected def textKey = "text"
```
This should probably be a `def`, not `val`, because defs for constants are actually more efficient (and easier to override than vals) – constants are stored in the constant section in the classfile and subsequently in the constant pool on the JVM heap, whenever a method returns a constant it will return an address from the constant pool - it will not allocate a new string. Now a `val` also makes a field in the class that stores the address of the constant, but this is superfluous since a method would already return a pointer to a string from the constant pool, a val would just inflate the in-memory size of the object with a copy of that pointer inside the _object_ (that is already stored in the _class_ after it's loaded)

Reply:
Fixed. Thanks for your detail explanation!